runtime/src/bank.rs

```diff
@@ -444,35 +445,113 @@ pub struct TransactionLogCollector {
     pub mentioned_address_map: HashMap<Pubkey, Vec<usize>>,
 }
 
-#[derive(Clone, Debug, Eq, PartialEq)]
-pub enum HashAgeKind {
-    Extant,
-    DurableNoncePartial(Pubkey, Account),
-    DurableNonceFull(Pubkey, Account, Option<Account>),
+pub trait NonceRollbackInfo {
```

Comment:
this looks better names to me, thanks for working so hard on renaming many things based on the new naming.